ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -113,7 +113,7 @@ ${listenerText}
 sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" ;\\
 sudo apt-get update && sudo apt-get install boundary ;\\
 boundary server -config="${configPath}/pki-worker.hcl"`;
-    const hcpContent = `wget -q https://releases.hashicorp.com/boundary-worker/0.11.2+hcp/boundary-worker_0.12.0+hcp_linux_amd64.zip ;\\
+    const hcpContent = `wget -q https://releases.hashicorp.com/boundary-worker_0.12.0+hcp_linux_amd64.zip ;\\
```

Comment:
```suggestion
    const hcpContent = `wget -q https://releases.hashicorp.com/boundary-worker/0.12.0+hcp/boundary-worker_0.12.0+hcp_linux_amd64.zip ;\\
```

We still need the `boundary-worker/0.12.0+hcp/` portion of the URL.

Reply:
@cameronperera dang, thought i had that forward slash in there